src/components/AddItem.js

```diff
@@ -114,8 +116,17 @@ const AddItem = () => {
           ></input>
           <label htmlFor="not-soon"> Not soon</label>
         </fieldset>
-        <button type="submit">Add an item</button>
+        <GreenButton
+          type="submit"
+          clickFunction={item && frequency && handleClick}
```

Comment:
Thank you again @sandaiiyahh for pointing out that you were able to add an empty item, or an item without a frequency to the list. I think that using the Material-UI button component messes with its functionality somehow. This ternary here for the `clickFunction` prop is a quick patch. It just checks to see that there is an input and a frequency selected (by checking if the state values exist), and then passing in the `handleClick` function in the prop if they do. So, as of now, the user can't submit an item that doesn't have both an item name _and_  a frequency selected. I don't _love_ this solution because I feel like the button should just work haha, but maybe for now it will do?

Reply:
Yup, awesome this works! 🎉 That's interesting the Material-UI does that. Being that there's no longer an indication or anything if a user tries to submit something blank, we should probably add an issue to have another Sweet Alert for when a user doesn't select anything (name and frequency)